support/Sources/GraphQL.swift

```diff
@@ -294,6 +298,38 @@ public struct SchemaViolationError: Error {
 	}
 }
 
+public enum InputValue<T> {
+    case some(T)
+    case null
+    case undefined
+    
+    public init(orNull optional: Optional<T>)  {
+        if let value = optional {
+            self = .some(value)
+        } else {
+            self = .null
+        }
+    }
+    
+    public init(orUndefined optional: Optional<T>)  {
+        if let value = optional {
+            self = .some(value)
+        } else {
+            self = .undefined
+        }
+    }
+}
+
+public extension Optional {
+    public var orNull: InputValue<Wrapped> {
+        return InputValue(orNull: self)
+    }
+    
+    public var orUndefined: InputValue<Wrapped> {
+        return InputValue(orUndefined: self)
+    }
```

Comment:
These optional extensions feel redundant to me. Not everyone will want to use them. These should be removed from the generator. Consumers that want this extension can always build it themselves.

Reply:
True. Agreed.